schemas/pe.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: pe
+  title: PE Header
+  group: 2
+  description: These fields contain Windows Portable Executable (PE) metadata.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - process
+  fields:
+
+    - name: original_file_name
+      format: string
+      level: extended
+      type: keyword
+      description: Internal name of the file, provided at compile-time.
+      example: MSPAINT.EXE
+
+
+    - name: file_version
+      level: extended
+      type: keyword
+      short: Process name.
+      description: Internal version of the file, provided at compile-time.
+      example: 6.3.9600.17415
+
+
+    - name: description
+      format: string
```

Comment:
This is not needed, please remove all instances of `format:string`.

The places you've seen this are on numeric fields where we don't want thousands separators (e.g. PID). It doesn't do much, it only passes this along to Beats which uses it when it generates Kibana index templates that set this field format attribute. This isn't something that Elasticsearch is even aware of.